src/main/java/de/fraunhofer/aisec/cpg/TranslationConfiguration.java

```diff
@@ -300,6 +300,7 @@ public Builder registerPass(@NonNull Pass pass) {
     public Builder defaultPasses() {
       registerPass(new FilenameMapper());
       registerPass(new TypeHierarchyResolver());
```

Comment:
Not necessarily related to this PR, but can we rename the `TypeHierarchyResolver` since it does not really resolve types at all?

Reply:
How do you mean? It resolves the hierarchy of object types in the graph by adding the necessary relationships to their respective RecordDeclarations

Reply:
@Masrepus I think he refers to the last Note in the initial description

Reply:
Ah I see, so renaming to something like RecordHierarchyResolver would probably be the way to go